CS342MemberViews/Sources/RajPabari/DescriptionView.swift

```diff
@@ -0,0 +1,39 @@
+//
+// This source file is part of the CS342 2023 Application project
+//
+// SPDX-FileCopyrightText: 2023 Stanford University
+//
+// SPDX-License-Identifier: MIT
+//
+import SwiftUI
+
+
+public struct DescriptionView: View {
```

Comment:
In the spirit of reuse, you can just use the View in the Instructor Views target instead of copy and pasting your view here.
Either that, or extend this view with a meaningful functional in addition to what is currently found in the InstructorViews target.

Reply:
@rajpabari I have clarified this in the assignment description 👍